tests/testthat/test-model-summary.R

```diff
@@ -211,10 +211,80 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
       ofvs <- model_summary(mod)[[OFV_COL]]
       ofv <- ofvs[[length(ofvs)]]
-      expect_identical(ofv[["ofv_no_constant"]], NA_real_)
-      expect_identical(ofv[["ofv_with_constant"]], NA_real_)
+      expect_identical(ofv[["ofv_no_constant"]], NA_integer_)
```

Comment:
This is discussed in another comment above, but I think we want to change this back.